src/sentry/api/endpoints/project_tagkey_values.py

```diff
@@ -51,5 +52,17 @@ def get(self, request, project, key):
             queryset=queryset,
             order_by='-last_seen',
             paginator_cls=DateTimePaginator,
-            on_results=lambda x: serialize(x, request.user),
+            on_results=lambda results: serialize(
+                map(  # XXX: This is a pretty big abstraction leak
```

Comment:
any reason why this uses `map()` syntax instead of list comprehensions like some of the other ones further down?